taca/analysis/analysis.py

```diff
@@ -132,8 +134,18 @@ def transfer_run(run_dir, analysis):
         # Maybe throw an exception if possible?
         logger.error("Trying to force a transfer of run {} but the sequencer was not recognized.".format(run_dir))
     else:
-        runObj.transfer_run("nosync", os.path.join(CONFIG['analysis']['status_dir'], 'transfer.tsv'),
+        runObj.transfer_run(os.path.join("nosync",CONFIG['analysis']['status_dir'], 'transfer.tsv'),
                             analysis) # do not start analsysis automatically if I force the transfer
+        #Send an email after the transfer to Irma 
+        runname =os.path.basename(os.path.normpath(run_dir))
```

Comment:
right, however you have in  your hands the object`runObj ` that should contain this information. It would be better if you can extract it from there